java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -566,6 +567,18 @@ void testReadParquet() {
     }
   }
 
+  @Test
+  void testReadParquetBinary() {
+    ParquetOptions opts = ParquetOptions.builder()
+        .includeColumn("value", true)
+        .build();
```

Comment:
This test would be better if it included a combination of read-as-string and read-as-binary columns.